inst/emails/contact_initial_aon

```diff
@@ -2,20 +2,25 @@ Dear %s,
 
 Congratulations on your new NSF Arctic Sciences award %s %s.
 
-The NSF-funded Arctic Data Center (https://arcticdata.io/) support team is passionate about Arctic science. We are here to support researchers like yourself in storing and sharing valuable data.
+The NSF-funded Arctic Data Center support team is here to for researchers like yourself in storing and sharing valuable data.
 
-We are contacting you today because projects funded by the NSF ARC Arctic Observing Network (AON) program have specific requirements for data and metadata storage. The Arctic Data Center is a long-lived, publicly accessible archive that was created to help researchers meet the NSF data and metadata storage requirements.
+We are contacting you today because projects funded by the NSF ARC Arctic Observing Network (AON) program have specific requirements for data and metadata storage. The Arctic Data Center is a long-lived, publicly accessible archive that was created to help researchers meet the storage requirements.
 
-At this time, please note that NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are required to be submitted within 6 months of collection. More detailed information about what will be expected of you can be found at https://arcticdata.io/submit/#who-must-submit.
-
-With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Typically, NSF researchers already do a great job of collecting data and metadata, but you will want to ensure your data and metadata are collected in a manner that can be easily accessed and understood by other researchers after it is published. We recommend that you visit our data submission guidelines page (https://arcticdata.io/submit/) to get a better understanding of what is expected for submissions to the Arctic Data Center.
-
-Please note that for AON program projects, NSF requires that:
-    - Real-time data be made publicly available immediately. If there is any question about what constitutes real-time data, please contact the appropriate NSF program officer.
+Please note that for AON program projects, NSF has the additional requirements:
+    - Real-time data be made publicly available immediately. 
```

Comment:
"If there is any question about what constitutes real-time data, please contact the appropriate NSF program officer." This was in there for a reason. The language can be softened, but they need to talk to program officers about what part of their data is real-time.